loopy/program.py

```diff
@@ -370,6 +384,9 @@ def __hash__(self):
         self.update_persistent_hash(key_hash, LoopyKeyBuilder())
         return hash(key_hash.digest())
 
+
+Program = TranslationUnit
```

Comment:
Can you inherit and make the constructor throw a warning with a 2022 removal?

Reply:
6151b27